web-local/config-ddl-local.yaml

```diff
@@ -29,6 +29,11 @@ systemOfRecord:
                table.audit: sys_audit
                table.delta: sys_delta
                table.history: sys_history
+            migration:
+               keyspace: app_global
+               table.audit: migration_audit
+               table.delta: migration_delta
```

Comment:
The idea of having a config file to specify table names confuses me. It gives the illusion of choice with table names, even though if it doesn't conform to the usual format, there will be a failure in the DeltaPlacementFactory